src/js/components/homepage/hero/TotalAmount.jsx

```diff
@@ -54,8 +56,14 @@ const TotalAmount = ({
         }
     }, [isLoading, total, completeIncrement]);
 
+    console.log('flag', IS_DYNAMIC_RENDERER);
```

Comment:
I guess this should never log to a user's console, but wondering if we should remove it anyway